server.js

```diff
@@ -1,27 +1,65 @@
 import express from "express";
 import cors from "cors";
+import boardgames from "./data/boardgames.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// API Documentation
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json({
+    message: "Welcome to the Jonas boardgames API! Choose from the endpoints below:",
+    endpoints: {
+      "/boardgames": "Explore all boardgames",
+      "/category?category=<category>": "Get all boardgames by category",
+      "/maxPlayers?maxPlayers=<number>": "Get all boardgames by maximum number of players",
+      "/boardgames/:id": "Get a single boardgames by ID"
+    }
+  });
+});
```

Comment:
[Express List Endpoints](https://www.npmjs.com/package/express-list-endpoints) is a great package so you don't have to update your docs manually